README.md

```diff
@@ -136,6 +136,45 @@ passing the instance to `testmod` or constructors of `DT*` objects. Defaults
 are provided, based on a long-term usage in SciPy.
 
 
+## The Scpdt Pytest Plugin
+
+The pytest plugin enables the use of scpdt tools to perform doctests. 
+
+Presently, pytest is configured to utilize scpdt's `DTChecker` for comparing the expected result (`want`) of test examples to the actual result (`got`) produced by the doctest runner.
+
+To utilize the pytest plugin, follow these steps:
+
+1. Install the scpdt tool
+
+Ensure you have the scpdt tool installed from GitHub. You can install it using pip with the following command:
+
+```bash
+pip install git+https://github.com/ev-br/scpdt.git@main
+```
+
+2. Register/Load the Plugin
+
+Next, you need to register or load the pytest plugin in either your test module or your `conftest.py` file. 
+
+To do this, add the following line of code:
```

Comment:
It's not immediately clear to me where to add it. A worked example would be best.

Reply:
What about adding a comment like this?
```python
# in your conftest.py file or test module

pytest_plugins = "scpdt"
```